src/Core/Systems/TextureSystem.cc

```diff
@@ -47,7 +47,9 @@ namespace esp
   std::shared_ptr<EspTexture> TextureSystem::acquire(const std::string& name)
```

Comment:
Wciąż w acquire nie mamy wpływu na parametry tekstury. Nie powinniśmy tej struktury przekazać w argumencie?

Reply:
Ok, nie wyjaśniłem tego do końca. Dodałem parametr ze strukturą do `load`, które nie sprawdza czy tekstura znajduje się w mapie, tylko bezwarunkowo ją ładuje. Zrobiłem tak, poniważ w sytuacji kiedy ładowalibyśmy teksturę za pomocą `acquire` i podalibyśmy jakieś parametry, to nie mielibyśmy wpływu na to czy tekstura zwrócona z mapy ma takie same parametry. Możnaby to sygnalizować, zwracać wtedy nulla, lub ignorować. Ja postanowiłem rozdzielić acquire i load, żeby uniknąć takich konfliktów.
Czujesz, że inny z tych sposobów jest lepszy?

Reply:
Ostatecznie jak uzgodniiliśmy parametry zostały dodane do metody `acquire` i w razie gdy nie zgadzają się z parametrami zwracanej tekstury jest wysyłany warning, ale tekstura jest zwracana mimo to.